src/js/components/search/SearchSidebar.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * SearchSidebar.jsx
+ * Created by Emily Gullo 10/14/2016
+ **/
+
+import React from 'react';
+import SearchOption from './SearchOption.jsx';
+
+export default class SearchSidebar extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        let options = [
```

Comment:
Bring this out of render() - if it's static, we don't need to reinitialize during every render